proxy/cassandra/src/graal/reflect-config.json

```diff
@@ -612,215 +601,13 @@
-    {"name":"getValuesBuilder","parameterTypes":["int"] }, 
-    {"name":"getValuesCount","parameterTypes":[] }, 
-    {"name":"getValuesList","parameterTypes":[] }, 
-    {"name":"setValues","parameterTypes":["int","com.google.protobuf.Value"] }
-  ]
-},
-{
-  "name":"com.google.protobuf.NullValue",
-  "methods":[
-    {"name":"getValueDescriptor","parameterTypes":[] }, 
-    {"name":"valueOf","parameterTypes":["com.google.protobuf.Descriptors$EnumValueDescriptor"] }
-  ]
-},
-{
-  "name":"com.google.protobuf.Struct",
-  "methods":[
-    {"name":"getDefaultInstance","parameterTypes":[] }, 
-    {"name":"newBuilder","parameterTypes":[] }
-  ]
-},
-{
-  "name":"com.google.protobuf.Value",
-  "methods":[
-    {"name":"getBoolValue","parameterTypes":[] }, 
-    {"name":"getKindCase","parameterTypes":[] }, 
-    {"name":"getListValue","parameterTypes":[] }, 
-    {"name":"getNullValue","parameterTypes":[] }, 
-    {"name":"getNullValueValue","parameterTypes":[] }, 
-    {"name":"getNumberValue","parameterTypes":[] }, 
-    {"name":"getStringValue","parameterTypes":[] }, 
-    {"name":"getStringValueBytes","parameterTypes":[] }, 
-    {"name":"getStructValue","parameterTypes":[] }, 
-    {"name":"newBuilder","parameterTypes":[] }
-  ]
-},
-{
-  "name":"com.google.protobuf.Value$Builder",
-  "methods":[
-    {"name":"clearBoolValue","parameterTypes":[] }, 
-    {"name":"clearKind","parameterTypes":[] }, 
-    {"name":"clearListValue","parameterTypes":[] }, 
-    {"name":"clearNullValue","parameterTypes":[] }, 
-    {"name":"clearNumberValue","parameterTypes":[] }, 
-    {"name":"clearStringValue","parameterTypes":[] }, 
-    {"name":"clearStructValue","parameterTypes":[] }, 
-    {"name":"getBoolValue","parameterTypes":[] }, 
-    {"name":"getKindCase","parameterTypes":[] }, 
-    {"name":"getListValue","parameterTypes":[] }, 
-    {"name":"getListValueBuilder","parameterTypes":[] }, 
-    {"name":"getNullValue","parameterTypes":[] }, 
-    {"name":"getNullValueValue","parameterTypes":[] }, 
-    {"name":"getNumberValue","parameterTypes":[] }, 
-    {"name":"getStringValue","parameterTypes":[] }, 
-    {"name":"getStringValueBytes","parameterTypes":[] }, 
-    {"name":"getStructValue","parameterTypes":[] }, 
-    {"name":"getStructValueBuilder","parameterTypes":[] }, 
-    {"name":"setBoolValue","parameterTypes":["boolean"] }, 
-    {"name":"setListValue","parameterTypes":["com.google.protobuf.ListValue"] }, 
-    {"name":"setNullValue","parameterTypes":["com.google.protobuf.NullValue"] }, 
-    {"name":"setNullValueValue","parameterTypes":["int"] }, 
-    {"name":"setNumberValue","parameterTypes":["double"] }, 
-    {"name":"setStringValue","parameterTypes":["java.lang.String"] }, 
-    {"name":"setStringValueBytes","parameterTypes":["com.google.protobuf.ByteString"] }, 
-    {"name":"setStructValue","parameterTypes":["com.google.protobuf.Struct"] }
-  ]
-},
-{
-  "name":"com.google.protobuf.any.Any",
-  "allDeclaredMethods":true,
```

Comment:
@dwijnand this needs to stay

Reply:
It's now in `proxy/core/src/main/resources/META-INF/native-image/io.cloudstate/cloudstate-proxy-core/cloudstate-reflect-config.json.conf`.